Source/Core/Core/IOS/USB/Emulated/Infinity.h

```diff
@@ -44,16 +44,15 @@ class InfinityUSB final : public Device
   int ChangeInterface(u8 interface) override;
   int GetNumberOfAltSettings(u8 interface) override;
   int SetAltSetting(u8 alt_setting) override;
-  int SubmitTransfer(std::unique_ptr<CtrlMessage> message) override;
+  int SubmitTransfer(EmulationKernel& ios, std::unique_ptr<CtrlMessage> message) override;
   int SubmitTransfer(std::unique_ptr<BulkMessage> message) override;
-  int SubmitTransfer(std::unique_ptr<IntrMessage> message) override;
+  int SubmitTransfer(EmulationKernel& ios, std::unique_ptr<IntrMessage> message) override;
```

Comment:
`CtrlMessage` and `IntrMessage` derive from `TransferCommand` which has a protected `EmulationKernel& m_ios;`. If you added a "getter" to `TransferCommand` you wouldn't need to modify this function prototype and have the redundant parameter.